docs/source/data-publishing/ogcapi-tiles.rst

```diff
@@ -130,6 +131,47 @@ Following code block shows how to configure pygeoapi to read Mapbox vector tiles
              name: pbf
              mimetype: application/vnd.mapbox-vector-tile
 
+MVT-postgresql
+^^^^^^^^^^^^^^
+
+.. note::
+   Requires Python packages sqlalchemy, geoalchemy2 and psycopg2-binary
+
+.. note::
+   Must have PostGIS installed with protobuf-c support 
+
+.. note::
+   Geometry must be using EPSG:4326
```

Comment:
I understand that this note is outdated for the postgresql provider. Is the note valid for this case?